recipe/meta.yaml

```
@@ -22,17 +23,20 @@ requirements:
     - wheel
   run:
     - python
-    - jinja2 >=2.10
-    - sphinx >=4.2
+    - sphinx >=6
+    - tabulate >=0.8.10
+    - tomli >=1.1.0  # [py311]
```

```suggestion
    - tomli >=1.1.0  # [py<311]
```
https://github.com/numpy/numpydoc/blob/v1.7.0/pyproject.toml#L31